backend/api/search.py

```diff
@@ -1,10 +1,25 @@
-from flask import request, jsonify, make_response, g, Blueprint
+from flask import request, jsonify, g, Blueprint
 from elasticsearch.exceptions import RequestError
 from flask_security import login_required
 
+
 search = Blueprint('api_projects_search', __name__)
 
 
+@search.route('/api/search/authors', methods=['GET'])
+@login_required
+def get_all_authors():
+    authors = g.projects.distinct('authors')
+    return jsonify(authors)
+
+
+@search.route('/api/search/tags', methods=['GET'])
+@login_required
+def get_all_tags():
+    tags = g.projects.disctinct('tags')
+    return jsonify(tags)
```

Comment:
I'm not sure, felt like it's more of a search thing but we can move it to projects